src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/ReceivedMessagesHealthController.java

```diff
@@ -114,6 +120,12 @@ private boolean isDateWithinWorkingHours(LocalDateTime localDateTime) {
     }
 
     private boolean isNotEnabledForEnvironment(String env) {
```

Comment:
Obviously we'd need a unit test to check this in various different scenarios.
I'd suggest making this method `public` too to make it easier to test in isolation.  Later on we can refactor it to be in it's own class.